esmvaltool/recipes/recipe_ocean_temperature_cmip6.yml

```diff
@@ -0,0 +1,84 @@
+# ESMValTool
+# recipe_ocean_temperature_cmip6.yml
+---
+documentation:
+  description: |
+    Recipe to demonstrate a basic plots based on the monthly ocean
+    temperature in CMIP6.
+
+    Note that the time series will not function without the FX files, which
+    are not available yet on JASMIN (2019/04/15).
+
+    The maps will be plotted in their models native grids, which can sometimes
+    look strange. Regridding is a possible solution, but will not work without
+    the fx files, which are not yet available.
+
+  authors:
+    - demo_le
+
+  maintainer:
+    - demo_le
+
+  references:
+    - demora2018gmd
+
+  projects:
+    - ukesm
+
+
+datasets:
+     # CMIP6
+     - {dataset: BCC-CSM2-MR,      project: CMIP6,    exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: BCC-ESM1,         project: CMIP6,    exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: CESM2,            project: CMIP6,    exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: CNRM-CM6-1,       project: CMIP6,    exp: historical, ensemble: r1i1p1f2, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: GISS-E2-1-G,      project: CMIP6,    exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: IPSL-CM6A-LR,     project: CMIP6,    exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: MIROC6,           project: CMIP6,    exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 2000,  end_year: 2014}
+     - {dataset: UKESM1-0-LL,      project: CMIP6,    exp: historical, ensemble: r1i1p1f2, grid: gn, start_year: 2000,  end_year: 2014}
+
+
+preprocessors:
+  # --------------------------------------------------
```

Comment:
Please remove commented out stuff

Reply:
I left the commented stuff out because it "should" work. However, the badc disk on JASMIN do not yet include the fx files for all models. When they are there, it will be important to test these functions with that data.

Reply:
Commented out stuff should not be merged, because it just makes a mess. It would be good if you could remove it and make a new pull request when new functions once this is possible.